packages/contracts-bedrock/test/L2/L2Genesis.t.sol

```diff
@@ -174,11 +174,15 @@ contract L2GenesisTest is Test {
         genesis.writeGenesisAllocs(_path);
 
         uint256 expected = 0;
-        expected += 2048 - 2; // predeploy proxies
-        expected += 21; // predeploy implementations (excl. legacy erc20-style eth and legacy message sender)
+        // predeploy proxies; WETH and GovernanceToken are not behind proxies
+        expected += Predeploys.PREDEPLOY_COUNT - 2;
+        // predeploy implementations (excl. legacy erc20-style eth and legacy message sender)
+        // setPredeployImplementations function in L2Genesis.s.sol sets 20 predeploy implementations (useInterop ==
+        // false)
+        expected += 20;
         expected += 256; // precompiles
-        expected += 13; // preinstalls
-        expected += 1; // 4788 deployer account
```

Comment:
Why is this `1` gone?

Reply:
Re-added the +1 and set useSoulGasToken to true to ensure SoulGasToken is included in the count.